backend/src/project/endpoints.ts

```diff
@@ -2,11 +2,27 @@ import { Request, Response } from "express";
 import { ObjectId } from "mongodb";
 import { getCollections } from "../app";
 import { validateProject } from "./service";
+import { User } from "../user/model";
 
-export const getAllProjects = async (_req: Request, res: Response) => {
+export const isAuthenticated = (req: Request): string => {
```

Comment:
Trochę niefortunnie nazwana funkcja, bo `is` sugeruje, że funkcja zwraca boola czy jest zalogowany czy nie. Lepsza będzie nazwa `getAuthenticatedUserName`. I da się ją uprościć:
`const user = req.user as User;`
`return user?.username`;